src/main/java/com/yurjinia/common/security/jwt/constants/JwtConstants.java

```diff
@@ -1,8 +1,14 @@
 package com.yurjinia.common.security.jwt.constants;
 
+import org.springframework.beans.factory.annotation.Value;
+
 public class JwtConstants {
+
     public static final String AUTHORIZATION_HEADER = "Authorization";
     public static final String TOKEN_PREFIX = "Bearer ";
     public static final String USERNAME_KEY = "sub";
-    public static final String AUTH_URL = "/api/v1/auth/";
+
+    @Value("${APP.AUTH.URL}")
+    public static String AUTH_URL;
```

Comment:
value fields can't be static, use them in those classes where want to inject them.